packages/cli/commands/execute.ts

```diff
@@ -17,26 +15,19 @@ import {
 	IWorkflowExecutionDataProcess,
 	LoadNodesAndCredentials,
 	NodeTypes,
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Let's keep them for now since it would change scope and also requires testing again. The person who is touching the file next time should fix that 👍🏼